### PR TITLE
adding classifications to pm bidders

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Dual Merge: [BE](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/497) ~& [Mock](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/174)~

~blocker: waiting on which classifications to show (there is also a Offer Handshake button that will be added to this table, so we are getting even tighter on real estate)~ Just gonna show them all for now 🤷

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
Not sure how to handle this. the ticket asks for the shortCode, which we use for Client Badges, but we do so via a static obj in `src/Components/BidderPortfolio/ClientBadge/ClientBadge.jsx`. Should I extract that obj to be more reusable? or just pop `tp_code` in the table?

**Comment by mjoyce91** [comment]
I know this is in progress, but just testing other stuff, I noticed the column headers aren't aligned

Screen Shot 2021-04-06 at 1 36 12 PM

**Comment by elizabeth-jimenez** [comment]
> I know this is in progress, but just testing other stuff, I noticed the column headers aren't aligned
> 
> Screen Shot 2021-04-06 at 1 36 12 PM

sshh

**Comment by mjoyce91** [comment]
Will also need to consider how we display these... tooltip?

Screen Shot 2021-04-07 at 11 56 28 AM

**Comment by elizabeth-jimenez** [comment]
> Will also need to consider how we display these... tooltip?
> 
> Screen Shot 2021-04-07 at 11 56 28 AM

oooo u can see them! I don't run my local mock together with the api, so i havent been able to see them go all the way through. I put a question up above https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/1430#issuecomment-814226273 (lol, next time i'll tag to make sure y'all see it) that is related to this

**Comment by mjoyce91** [comment]
> ### Question
> Not sure how to handle this. the [ticket](https://metaphase.atlassian.net/browse/TM-2145) asks for the shortCode, which we use for Client Badges, but we do so via a local obj in `src/Components/BidderPortfolio/ClientBadge/ClientBadge.jsx`. Should I extract that obj to be more reusable? or just pop `tp_code` in the table?

I'd go with abstracting that `icons` object you're referencing to be more reusable. However, looking at the designs, I'm still concerned that it doesn't really capture how many classifications a user can have. I think we need to dig into the design further.

**Comment by elizabeth-jimenez** [comment]
> > ### Question
> > Not sure how to handle this. the [ticket](https://metaphase.atlassian.net/browse/TM-2145) asks for the shortCode, which we use for Client Badges, but we do so via a local obj in `src/Components/BidderPortfolio/ClientBadge/ClientBadge.jsx`. Should I extract that obj to be more reusable? or just pop `tp_code` in the table?
> 
> I'd go with abstracting that `icons` object you're referencing to be more reusable. However, looking at the designs, I'm still concerned that it doesn't really capture how many classifications a user can have. I think we need to dig into the design further.

ok, i'll do that and make sure client badges component can still use it, and use it here, but I agree, it still wouldnt look good with 7+ classifications

File: src/sass/_bureau.scss [hunk 1]
```diff
@@ -366,7 +366,6 @@
 }
 
 .position-manager-bidders {
-  margin-top: 50px;
```
Comment (elizabeth-jimenez) [inline src/sass/_bureau.scss:369]: A lot of the css in this file was deprecated as newer (and lower in the file) styles were added and some elements probably were no longer being used(like H1). Everything should look the same or a little better 🤞

File: src/Components/BureauPage/PositionManagerBidders/PositionManagerBidders.jsx [hunk 2]
```diff
@@ -18,6 +19,32 @@ import ShortListLock from '../ShortListLock';
 import MailToButton from '../../MailToButton';
 import { tertiaryCoolBlueLight, tertiaryCoolBlueLightest } from '../../../sass/sass-vars/variables';
 
+const getClassificationsInfo = (userClassifications, refClassifications) => {
```
Comment (elizabeth-jimenez) [inline src/Components/BureauPage/PositionManagerBidders/PositionManagerBidders.jsx:22]: 🤮  Would love a cleaner alternative

**Comment by elizabeth-jimenez** [comment]
image
image
image
image

**Comment by elizabeth-jimenez** [comment]
> I know this is in progress, but just testing other stuff, I noticed the column headers aren't aligned
> 
> Screen Shot 2021-04-06 at 1 36 12 PM

i've either corrected this in this PR or the SL feature one; the problem was two things:
1️⃣  different padding amounts on `th` vs `td`
2️⃣  the scroll bar `30px` on body

File: src/Components/BureauPage/PositionManagerBidders/PositionManagerBidders.jsx (hunk 2 above)
Comment (elizabeth-jimenez) [reply src/Components/BureauPage/PositionManagerBidders/PositionManagerBidders.jsx:23]: Mike helped me eliminate one for loop 🦜

File: src/sass/_bureau.scss [hunk 3]
```diff
@@ -432,6 +432,36 @@
...
+    .arrow-regular {
+      border: none;
+    }
+    .classification-wrapper {
+      padding-bottom: 10px;
+      .classification-icon {
+        color: $tertiary-success;
+        font-size: 20px;
+        padding-right: 10px;
+        min-width: 30px;
```
Comment (elizabeth-jimenez) [inline src/sass/_bureau.scss:452]: `min-width` jic no icon comes through

**Comment by mjoyce91** [comment]
`height: fit-content` makes this look a little better on hover

Before:
Screen Shot 2021-05-25 at 10 07 55 AM

After:
Screen Shot 2021-05-25 at 10 07 48 AM

**Comment by elizabeth-jimenez** [comment]
> `height: fit-content` makes this look a little better on hover
> 
> Before:
> Screen Shot 2021-05-25 at 10 07 55 AM
> 
> After:
> Screen Shot 2021-05-25 at 10 07 48 AM

👍  never even noticed it was so off

**Comment by mjoyce91** [comment]
> `happy mike?`

😃

**Review by mjoyce91 — APPROVED** [review]

**Pull request merged**